App.tsx

```diff
@@ -110,7 +109,7 @@ const BaseApp: React.FunctionComponent<{
 }> = ({staging, setStaging}) => {
   const [avalancheCenterId, setAvalancheCenterId] = React.useState(Constants.expoConfig.extra.avalanche_center as AvalancheCenterID);
   const [date] = React.useState<Date>(defaultDate);
-  const dateString = apiDateString(date);
+  const dateString = date.toISOString();
```

Comment:
this is a different string format than what all the other routes are using, is that problematic? or are things generally working and the follow up PRs will resolve this?

Reply:
I mean the react navigation `dateString` field used internally

Reply:
As far as I was able to tell, this one line is the source of truth for every one of those `dateString` values, no?

Reply:
I would say no, not exactly! The `dateString` params get parsed into Date objects when a screen is entered, so that most downstream/child components deal with Date objects internally. 

When a child component decides we have to navigate somewhere else (in response to a user action or whatever), the current date is converted back to a string for react-navigation purposes. 

This was all done in #84